gtk/src/toga_gtk/fonts.py

```diff
@@ -58,7 +58,10 @@ def __init__(self, interface):
                         # Debian Buster doesn't include the typelib file in any package,
                         # but it works even without a cache_clear, so continue with a
                         # warning.
-                        warn(import_error.format("PangoFc"))
+                        warn(
+                            import_error.format("PangoFc"),
+                            stacklevel=2,
+                        )
```

Comment:
As above - we only need to split the line if it doesn't fit on a single line.
```suggestion
                        warn(import_error.format("PangoFc"), stacklevel=2)
```